Integrations/VulnDB/VulnDB.yml

```diff
@@ -209,7 +209,7 @@ script:
       secret: false
     - default: false
       defaultValue: '20'
-      description: Maximum number of entries to return. A high number of entries might effect performance.
+      description: Maximum number of entries to return. A high number of entries might affect performance.
```

Comment:
Maximum number of entries to return. A high number of entries might affect performance. The default value is "20".